projects/eventonica/README.md

```diff
@@ -14,8 +14,8 @@ You'll work on this project over the next 5 weeks. As you learn more web develop
 1. [Part 3 - jQuery UI](./eventonica-part3-jquery-ui.md)
 1. [Part 4 - API's](./eventonica-part4-apis.md)
 1. [Part 5 - Express Backend](./eventonica-part5-express-backend.md)
-1. [Part 6 - Postgres Database](./eventonica-part6-postgres.md)
-1. [Part 7 - React App](./eventonica-part7-react.md)<br>
+1. [Part 6 - Postgres Database](./eventonica-part6-postgres.md)</br>
+1. [Part 7 - React App](./eventonica-part7-react.md)</br>
```

Comment:
Don't have to add </br> tag. Just give simple new line.
Like this:
1. [Part 6 - Postgres Database](./eventonica-part6-postgres.md)
1. [Part 7 - React App](./eventonica-part7-react.md)

### High Level Overview